Gopkg.toml

```diff
@@ -34,7 +34,3 @@
 [[constraint]]
   branch = "master"
   name = "github.com/Masterminds/kitt"
-
```

Comment:
github.com/bacongobbler/browser should be in here, since it's a direct dependency.

Reply:
good catch

Reply:
done